MovieSharing/VideoAPIService.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  VideoRequest.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 29.09.21.
+//
+
+import Foundation
```

Comment:
- [ ] This is a bit larger comment. Please apply the logic to all your files though. This is the last main code style convention thing missing. We use comment mark seperators to visually seperate the different code parts.

Please try to add
// MARK: - Types
// MARK: - Properties
// MARK: - Public API
// MARK: - Private API

above the respective parts in your code. In doing that watch two things
- before and after a MARK leave an empty line
- only use the marks you need, so for example in your structs you currently only would need the Properties MARK

* Hint: Types are structs, classes etc you define within another class. So in the case of your VideoAPIService, VideoError should be defined within it.